google/datalab/bigquery/_table.py

```diff
@@ -388,7 +388,7 @@ def _init_job_from_response(self, response):
       job = _job.Job(job_id=response['jobReference']['jobId'], context=self._context)
     return job
 
-  def extract_async(self, destination, format='csv', csv_delimiter=',', csv_header=True,
+  def extract_async(self, destination, format='csv', csv_delimiter=None, csv_header=True,
```

Comment:
The docstring is still correct, the default is `","` only in the case of csv exports.

I don't think we should do opaque changes, the user should be responsible for specifying a bad parameter.